app/helpers/gov_uk_form_helper.rb

```diff
@@ -54,7 +54,7 @@ def govuk_error_message(text, args = {})
     content_tag :span, text, merge_with_class(args, 'govuk-error-message')
   end
 
-  def date_input_fields(prefix:, field_name:, form:, width: 'two-thirds', set_error_class_here: true, label: nil)
+  def date_input_fields(prefix:, field_name:, form:, width: 'two-thirds', set_error_class_here: true, label: nil, options: nil) # rubocop:disable Metrics/ParameterLists
```

Comment:
Rather than overriding in code, can this be added as an exclude to the .rubocop file?
Or investigate whether setting CountKeywordArgs to true might help?
https://docs.rubocop.org/rubocop/cops_metrics.html#metricsparameterlists